src/app/filters/displayDate.filter.js

```diff
@@ -0,0 +1,17 @@
+export default function (configuration, moment) {
+    'ngInject';
+
+    return function (date) {
+        var momentDate = moment(date);
+        var now = moment();
+
+        if (now.diff(momentDate, 'days') > 1) {
+            if (configuration.defaultLocale === 'en-EU') {
```

Comment:
Please provide explicit support for British English / UK locale since most people looking to localize will use the more specific locale than the EU-based one.

Code might look like this:
if (configuration.defaultLocale === 'en-GB' || configuration.defaultLocal === 'en-EU')